cypress/e2e/placeholder_does_not_show_after_clicking_dockument.cy.js

```diff
@@ -0,0 +1,17 @@
+describe("its going to test if the typo is going to change its value once the dokument is clicked", () => {
+    beforeEach(() => {
+        cy.visit("http://localhost:4200/");
+    });
+
+    it("will click on dokument and on shkruaj again", () => {
+        
+        cy.get("#editor > p > .typo").should("not.exist");
+        cy.get('[data-test="editor"]').type("pra asd e kaq ");
+        cy.wait(2000);
+        cy.get('button[id="uploadDocumentToggleButton"]').click();
+        cy.get('button[id="writeTextToggleButton"]').click();
+        cy.get('div>#editor-placeholder').should("not.exist");
+        cy.get('[data-test="editor"]').clear();
```

Comment:
Remove the empty line 7.
I believe the two second wait in line 10 is redundant in this context.
I've added a data-test attribute for the placeholder, feel free to use that as in `'[data-test="editor-placeholder"]'` in line 13. See best practices for selecting elements, https://docs.cypress.io/guides/references/best-practices#Selecting-Elements